Packs/QRadar/Integrations/QRadar_v2/QRadar_v2.py

```diff
@@ -805,12 +811,18 @@ def perform_offense_events_enrichment(
     events_query = {"headers": "", "query_expression": query_expression}
     print_debug_msg(f'Starting events fetch for offense {offense["id"]}.', client.lock)
     try:
-        query_status, search_id = try_create_search_with_retry(
-            client, events_query, offense
-        )
-        offense["events"] = try_poll_offense_events_with_retry(
-            client, offense["id"], query_status, search_id
-        )
+        for i in range(MAX_FETCH_EVENT_RETIRES):
```

Comment:
in case the functions raise an exception, do we want to keep trying?
if so, then the `try-except` block should be inside the loop.
```python
for i in range(MAX_FETCH_EVENT_RETIRES):
    try:
        ...
    except Exception as e:
        ...
```